nucleus-core/src/main/java/io/github/nucleuspowered/nucleus/modules/teleport/commands/TeleportAskHereCommand.java

```diff
@@ -65,19 +67,19 @@
         return new CommandElement[] {
             GenericArguments.flags()
                     .permissionFlag(TeleportPermissions.TELEPORT_HERE_FORCE, "f")
-                    .buildWith(NucleusParameters.ONE_PLAYER.get(serviceCollection))
+                    .buildWith(NucleusParameters.ONE_USER.get(serviceCollection))
```

Comment:
Why? How can you send a request to a player that's offline?

Reply:
Just make other plugins can handle this with custom logic. Maybe you want them implement `tpa`s themselves?

Reply:
No, `/tpa` is a player-player interaction and a plugin shouldn't be deciding for them.